src/ui/mod.rs

```diff
@@ -349,13 +357,7 @@ impl Ui {
                     .clicked()
                 {
                     event_proxy.send_event(UserEvent::RequestRedraw);
-                }
-                if state.can_capture {
-                    if ui
-                        .button(icon_to_char(Icon::ScreenshotMonitor).to_string())
-                        .on_hover_text(fl!("control_capture"))
-                        .clicked()
-                    {
+                    if state.can_capture {
```

Comment:
Screen capturing is not triggered by play button. The capture button should not be removed if there is not specific reasons.